### DISCUSSION TITLE
Learn NEAR CN Translation Project

### DISCUSSION THREAD

**Discussion by think-in-universe** [near-x/learn-near-community#8] [General]
Hey everyone, 

Please claim the tasks that you'd like to contribute to for Learn NEAR CN translation. 
To claim a task, please reply with the task you'd like to contribute, either as a translator or review is fine :) 

Please also engage with us on NEAR forum if you'd like to share any suggestions: https://gov.near.org/t/translate-learn-club-post-into-chinese/3098

The current translation workflow is as below:

1. Claim Task by replying in the corresponding threads
2. Start translating or help with review when article is translated
3. With at least two reviewers confirmed the quality of the post, submit a SputnikDAO proposal to receive payout to `learn-near-chian.spunikdao.near`
4. The payout will be shared between collaborators depending on their contribution

## Current Contributors

We need to know your NEAR account, so we'll be able to add you into Learn NEAR China DAO, and send payout to your account based on your contribution.

GitHub | NEAR
--- | ---
@marco-sundsk | sunhao.near
@MartinYeung5 | martinyeung.near
@shepf  | docker.near
@think-in-universe | robertyan.near
@tigerjibo | jibo.near
@wongxf | wongxf.near

## Guides

https://learnnear.club/guides/

Post | Translators | Reviewers | Status | PR | Payout
--- | --- | --- | --- | --- | ---
NEAR Account – Express Guide | @shepf | @think-in-universe | [Published](https://learnnear.club/zh/near%e8%b4%a6%e6%88%b7-%e5%bf%ab%e9%80%9f%e6%8c%87%e5%8d%97/) | #3 | [17](https://www.sputnik.fund/#/dao/learnnear.sputnikdao.near/proposals/14)
What is NEAR protocol? (2249 + 1463 words) | @marco-sundsk / @MartinYeung5   | @shepf / @wongxf  | [Published](https://learnnear.club/zh/near-ru-men/) | #20 + #22 | 67 + 43
NEAR 🤝 Ethereum (2738 words) | @wongxf  | @tigerjibo / @MartinYeung5   | [Published](https://learnnear.club/zh/near-%f0%9f%a4%9d-yi-tai-fang/) | #16 | 81
NEAR Protocol and Solana (1378 words) | @shepf | @tigerjibo / @MartinYeung5  | [Published](https://learnnear.club/zh/near-protocol-he-solana/) | #5 | 41
How the NEAR Rainbow Bridge works (1511 words) | @marco-sundsk | @wongxf / @shepf / @MartinYeung5   | [Published](https://learnnear.club/zh/near-cai-hong-qiao-shi-ru-he-gong-zuo-de/) | #7 | 45
How to Build on NEAR? Starting Guide | @shepf | @marco-sundsk / @think-in-universe | Under Review | #4 | 
NEAR Sputnik DAO: Empowering Communities (1435 words) | @shepf | @marco-sundsk  | Proposal Submitted | #6 | 43
Top 3 NFT Projects On The NEAR Protocol (2346 words) | @MartinYeung5 | @marco-sundsk  | [Published](https://learnnear.club/zh/near-xie-yi-shang-pai-ming-qian-san-de-you-xiu-nft-xiang-mu/) | #23 | 70

### NEAR Certified Developer Level 1

https://learnnear.club/courses/near-certified-developer-level-1/ 

Step | Translators | Reviewers | Status | PR | Word Count | Payout
--- | --- | --- | --- | --- | --- | ---
Course Intro  | @wongxf  | @think-in-universe  | Under Review | #18 | 258
Intro – NEAR Certified Developer Level 1  | @wongxf  | @think-in-universe | Merged | #19 | 1018
Step 1 – Perceiving web3 – NEAR Certified Developer Level 1  | @tigerjibo  | @shepf  | Under Review | #12 | 278
Step 2 – Reading web3 NEAR Certified Developer Level 1  | @shepf  | @tigerjibo / @MartinYeung5   | Under Review | #10 | 460
Step 3 – Writing web3 NEAR Certified Developer Level 1  | @shepf  | @tigerjibo | Merged | #11 | 831
Step 4 – Testing web3 NEAR Certified Developer Level 1  | @shepf  | @tigerjibo | Merged | #17 | 550
Step 5 – Deploying web3 NEAR Certified Developer Level 1  | @shepf  | -- | Merged | #21 | 603
Step 6 – Demo web3 NEAR Certified Developer Level 1  | @wongxf  | @danielwpz  | Merged | #29 | 62
Further Steps after NEAR Certified Developer Level 1  | @wongxf  | @danielwpz  | Merged | #28 | 103

## New Guides Posts

1. What is Octopus Network? https://learnnear.club/what-is-octopus-network/
2. How to build a Play To Earn game on NEAR Protocol? https://learnnear.club/how-to-build-a-play-to-earn-game-on-near-protocol/
3. A Quick Introduction To The DeBio Network https://learnnear.club/a-quick-introduction-to-the-debio-network/
4. Five New Frontiers for NFTs: https://medium.com/electric-capital/5-new-frontiers-for-nfts-c77de367631d

**Comment by think-in-universe** [comment]
如果你想参与 [Guides](https://learnnear.club/guides/) 系列文章的翻译或者审阅，请在这里回复，回复格式

`章节名 / 角色 (translator 或者 reviewer）/ 预计翻译完成日期（仅针对翻译）`， 例如 NEAR 🤝 Ethereum / reviewer

**Reply by Notdoufu** [reply]
How the NEAR Rainbow Bridge works / reviewer
What is NEAR protocol? / reviewer 
NEAR 🤝 Ethereum / translator

**Reply by marco-sundsk** [reply]
NEAR Sputnik DAO: Empowering Communities / reviewer
What is NEAR protocol? / translator / 2021-07-15

**Reply by tigerjibo** [reply]
NEAR 🤝 Ethereum    /reviewer    /2021-07-06
NEAR Protocol and Solana  /reviewer /review finish  /2021-07-03
NEAR Sputnik DAO: Empowering Communities  /reviewer /2021-07-04

**Reply by shepf** [reply]
How the NEAR Rainbow Bridge works / reviewer  /2021-07-03
What is NEAR protocol? / reviewer / 2021-07-8

**Reply by MartinYeung5** [reply]
How the NEAR Rainbow Bridge works / reviewer /2021-07-03
What is NEAR protocol? (1-6) / translator / 2021-07-06
Near-Ethereum / reviewer /2021-07-07
Near protocol and solana / reviewer /2021-07-08
How to build on near starting guid / reviewer /2021-07-12
What is NEAR protocol? 2 / reviewer /2021-07-14
Top 3 NFT Projects On The NEAR Protoco l/ translator / 2021-07-20

**Comment by think-in-universe** [comment]
如果你想参与 [NEAR Certified Developer Level 1](https://learnnear.club/courses/near-certified-developer-level-1/) 的翻译或者审阅，请在这里回复，回复格式

`章节名 / 角色 (translator 或者 reviewer）/ 预计翻译完成日期（仅针对翻译）`， 例如 Course Intro / translator / July 20th

**Reply by Notdoufu** [reply]
Intro – NEAR Certified Developer Level 1 / translator
Step 6 – Demo web3 NEAR Certified Developer Level 1  / translator
Further Steps after NEAR Certified Developer Level 1  / translator

**Reply by tigerjibo** [reply]
NEAR Certified Developer Step 1 – Perceiving web3 – NEAR Certified Developer Level 1 translator 2021-07-3
Step 2 – Reading web3 NEAR Certified Developer Level 1 / reviewer / 2021-07-10
Step 3 – Writing web3 NEAR Certified Developer Level 1 / reviewer / 2021-07-10
Step 4 – Testing web3 NEAR Certified Developer Level 1 / reviewer / 2021-07-10

**Reply by shepf** [reply]
Step 1 – Perceiving web3 – NEAR Certified Developer Level 1 / reviewer / 2021-07-4
Step 2 – Reading web3 NEAR Certified Developer Level 1  / translator / 2021-07-3
Step 3 – Writing web3 NEAR Certified Developer Level 1  / translator / 2021-07-3
Step 4 – Testing web3 NEAR Certified Developer Level 1 / translator / 2021-07-5
Step 5 – Deploying web3 NEAR Certified Developer Level 1 / translator / 2021-07-8

**Reply by MartinYeung5** [reply]
Step 2 – Reading web3 NEAR Certified Developer Level 1 / reviewer /2021-07-14
Step 3 – writing web3 near certified developer level 1 / reviewer /2021-07-29
Step 4 – Testing web3 NEAR Certified Developer Level 1 / reviewer / 2021-07-29
Step 5 – Deploying web3 NEAR Certified Developer Level 1 / reviewer / 2021-07-30

**Reply by Notdoufu** [reply]
Step 6 – Demo web3 NEAR Certified Developer Level 1  / translator
Further Steps after NEAR Certified Developer Level 1    / translator

**Comment by think-in-universe** [comment]
关于文章翻译的报酬（Payout），这里主要针对 支付方式、单片文章报酬、报酬分配方式提供了一些想法，欢迎讨论并提出建议。

### 支付方式

1. 每篇文章或教程翻译的 payout 请求将通过 DAO 提案的方式提交，由 [learnnear DAO](https://www.sputnik.fund/#/dao/learnnear.sputnikdao.near) 支付，例如  https://www.sputnik.fund/#/dao/learnnear.sputnikdao.near/proposals/9
2. 支付的接受者为 Learn NEAR CN DAO: https://www.sputnik.fund/#/dao/learn-near-china.sputnikdao.near
3. 然后每人的报酬将根据具体的贡献比例，在翻译者和审阅者之间进行分配，具体见**报酬分配**部分

### 单篇文章报酬

1. 按照每个英文单词 0.4元 RMB 计算，转换为对应的 NEAR 数量（参考价格：https://fanyi-pro.baidu.com/price ）
2. 源代码等不翻译的部分不计算在内

### 报酬分配

1. 通常而言，由于翻译所花时间较多，应该占据分配的大部分，但考虑到不同译者的翻译质量差异，审阅校对的工作量不同。
2. 审阅者可以根据翻译质量和审阅修改所花费的精力，对翻译质量评级（优秀 / 一般 / 较差），译者和审阅者需对评级达成一致。根据翻译质量，将调整**译者**和**审阅者**的分配比例，如
  - 优秀：80 / 20
  - 一般：60 / 40
  - 较差：40 / 60
3. 在 Learn NEAR DAO 收到相应报酬后，将根据比例，分别给译者和审阅者发放对应的报酬，将通过发起相应提案进行支付
4. 在 Learn NEAR CN DAO Council 投票通过提案后，Pyaout 将成功发放给对应的译者和审阅者
5. 如在报酬分配中间遇到的纠纷，Learn NEAR CN DAO 有最终裁定权

**Reply by marco-sundsk** [reply]
建议加一个，报酬分配中间遇到的纠纷，Learn NEAR CN DAO 有最终裁定权。

**Reply by think-in-universe** [reply]
@marco-sundsk cool，加了

**Reply by think-in-universe** [reply]
One proposal has been submitted for Guides posts: https://www.sputnik.fund/#/dao/learnnear.sputnikdao.near/proposals/26

**Comment by think-in-universe** [comment]
## About Learn NEAR CN DAO 

The SputnikDAO profile of the DAO is: https://www.sputnik.fund/#/dao/learn-near-china.sputnikdao.near

### How to join the DAO?

All the contributors of Learn NEAR CN project will be eligible to be proposed as council members of Learn NEAR CN DAO. After approval by the current council via proposal, new council members will be added. 

### The responsibility of the DAO

In the current stage, the council will be in charge of managing the treasury and membership of the organization, and can submit proposals to send payout to contributors of the DAO

**Reply by marco-sundsk** [reply]
I suggest not all contributors can become council members, we have to take the efficeincy into considerration, member count of council group ( which has the responsibility to vote) would better to be limitted in 7 even 5.  
Other contributors can take part in the DAO event through online groups such as wechat, telegram and gov.near.org.

**Reply by think-in-universe** [reply]
thanks Marco. yeah. I agree. When the number of the members increases, we're not able to ask everyone to vote as council member. 

I actually mean everyone can submit a `NewCouncil` proposal in https://www.sputnik.fund/#/dao/learn-near-china.sputnikdao.near as long as they make contribution to the DAO.

**Comment by almal** [comment]
Any updates on the progress?

**Reply by think-in-universe** [reply]
Thanks for asking Sasha. It's mainly blocked on my side for final review and merging the PR. We should be able to wrap up and merge all the current PRs in this week.

**Comment by think-in-universe** [comment]
## NEAR Related Glossary (English <-> Chinese)

Please check out the glossary here: https://github.com/near-x/learn-near-community/tree/master/cn/resources/glossary

Please feel free to submit PRs if you have disagreement or questions about the glossary.

**Comment by danielwpz** [comment]
@think-in-universe @tigerjibo @almal @shepf @MartinYeung5 
hey guys, do u think we shall use a kanban board for tracking this project? I often found it's hard to tell which article is still under translation and which needs further review and I have to click through each PR to see its progress. Maybe a kanban could help this I guess.
please feel free to share your thoughts

**Reply by think-in-universe** [reply]
sure. let's do this next time, when we have another 5+ or 10+ posts to translate.

**Reply by danielwpz** [reply]
👍

**Reply by MartinYeung5** [reply]
agree, it may good for the whole process. 
which "kanban" will be used as your suggestion?

**Comment by danielwpz** [comment]
token你们觉得翻译成代币还是通证比较好呢？

**Reply by danielwpz** [reply]
目前我看都是叫通证

**Reply by Notdoufu** [reply]
代币

**Reply by think-in-universe** [reply]
我们 glossary 建议用通证：https://github.com/near-x/learn-near-community/tree/master/cn/resources/glossary 

个人也建议用通证，比代币准确，而且“代币”的说法在国内有额外风险

**Reply by MartinYeung5** [reply]
我也覺得用通证會更好，避免不必要的風險

**Comment by danielwpz** [comment]
## NEAR Certified Developer

### NEAR Certified Developer Level 1

https://learnnear.club/courses/near-certified-developer-level-1/ 

Step | Translators | Reviewers | Status | PR | Word Count | Payout
--- | --- | --- | --- | --- | --- | ---
Course Intro  | @wongxf  | @think-in-universe  | Under Review | #18 | 258 | 3.57
Intro – NEAR Certified Developer Level 1  | @wongxf  | @think-in-universe | Merged | #19 | 1018 | 14.09
Step 1 – Perceiving web3 – NEAR Certified Developer Level 1  | @tigerjibo  | @shepf  | Under Review | #12 | 278 | 3.85
Step 2 – Reading web3 NEAR Certified Developer Level 1  | @shepf  | @tigerjibo / @MartinYeung5   | Under Review | #10 | 460 | 6.37
Step 3 – Writing web3 NEAR Certified Developer Level 1  | @shepf  | @danielwpz  | Merged | #11 | 831 | 11.50
Step 4 – Testing web3 NEAR Certified Developer Level 1  | @shepf  | @danielwpz  | Merged | #17 | 550 | 7.61
Step 5 – Deploying web3 NEAR Certified Developer Level 1  | @shepf  | @danielwpz  | Merged | #21 | 603 | 8.35
Step 6 – Demo web3 NEAR Certified Developer Level 1  | @wongxf  | @danielwpz  | Merged | #29 | 62 | 0.86
Further Steps after NEAR Certified Developer Level 1  | @wongxf  | @danielwpz  | Merged | #28 | 103 | 1.43

> Payout calculated by 1 NEAR = 28.9 CNY (as of 2021-08-19 13:00)

**Reply by danielwpz** [reply]
@tigerjibo 你有review #11 么？

**Reply by danielwpz** [reply]
#17 review的也相对较少看起来

**Reply by think-in-universe** [reply]
是的，是否 review 已实际的 comments 和 suggestions 为准

**Reply by shepf** [reply]
good job！

**Comment by Notdoufu** [comment]
<html>
<body>


Post | --|--|Contribution Percentage | --|-- |-- |-- 
-- |-- | -- |---- |  ---- | ---- | ---- | ---|
  |       | @shepf | @think-in-universe |@martinyeung5|@danielwpz |@wongxf|@marco-sundsk |@tigerjibo 
NEAR Account – Express Guide #3 | 60%|40%|--|--|--|--|--
What is NEAR protocol?   #20 (2249  words) | 2%|--|80%|18%|--|--|--
What is NEAR protocol?   #22 (1463 words) |  --|8%|12%|--|--|80%|--
NEAR 🤝 Ethereum (2738 words)  #16 | --|--|40%|20%|40%|--|--
NEAR Protocol and Solana (1378 words)  #5 | 60%|-|16%|8%|--|--|16%
How the NEAR Rainbow Bridge works (1511 words) #7 | 5%|5%|5%|--|5%|80%|--
How to Build on NEAR? Starting Guide #4 | --|--|--|--|--|--|-- 
NEAR Sputnik DAO: Empowering Communities (1435 words) #6 | 35%|--|--|--|--|60%|5%
Top 3 NFT Projects On The NEAR Protocol (2346 words) #23 |--|--|80%|--|--|20%|-- 
Total  |53.44|12.49|155.97|31.54|34.65|110.2|8.71

</body>
</html>


<html>
<body>


Step | --|--|Contribution Percentage | --|-- |-- |-- 
-- |-- | -- |---- |  ---- | ---- | ---- | ---|
  |       | @shepf | @think-in-universe |@martinyeung5|@danielwpz |@wongxf|@marco-sundsk |@tigerjibo 
Course Intro #18 --|40%|--|--|60%|--|--
Intro – NEAR Certified Developer Level 1  #19 | --|10%|--|30%|60%|--|--
Step 1 – Perceiving web3 – NEAR Certified Developer Level 1 #12 | 20%|20%|--|--|--|--|60%
Step 2 – Reading web3 NEAR Certified Developer Level 1  #10 | 60%|10%|10%|10%|--|--|10%
Step 3 – Writing web3 NEAR Certified Developer Level 1   #11 | 80%|4%|6%|10%|--|--
Step 4 – Testing web3 NEAR Certified Developer Level 1   #17 | -80%|2%|4%|10%|--|--|4%
Step 5 – Deploying web3 NEAR Certified Developer Level #21 | 80%|--|7%|13%|--|--
Step 6 – Demo web3 NEAR Certified Developer Level 1  #29 |  --|--|--|20%|80%|--|--
Further Steps after NEAR Certified Developer Level 1  #28 | --|--|--|20%|80%|--|--
Total  |25.56|4.8562|2.2159|8.3186|12.428|--|3.2514

</body>
</html>

**Comment by think-in-universe** [comment]
### New Guides Posts

1. What is Octopus Network? https://learnnear.club/what-is-octopus-network/
2. How to build a Play To Earn game on NEAR Protocol? https://learnnear.club/how-to-build-a-play-to-earn-game-on-near-protocol/
3. A Quick Introduction To The DeBio Network https://learnnear.club/a-quick-introduction-to-the-debio-network/
4. Five New Frontiers for NFTs: https://medium.com/electric-capital/5-new-frontiers-for-nfts-c77de367631d

**Reply by MartinYeung5** [reply]
Five New Frontiers for NFTs/ translator/ 2021-09-30
How to build a Play To Earn game on NEAR Protocol?/ translator/ 2021-10-10

**Reply by Notdoufu** [reply]
What is Octopus Network? /translator /2021-09-30

**Reply by jzhang19** [reply]
A Quick Introduction To The DeBio Network / translator / 2022-01-22
